specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2021-10-01/networksecurityperimeter.json

```diff
@@ -0,0 +1,433 @@
+{
+  "swagger": "2.0",
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "info": {
+    "version": "2021-10-01",
+    "title": "Network security perimeter common type definitions",
+    "description": "Common types for network security perimeters based on a shared API specification. These common, versioned type definitions are intended for resource providers (RPs, except Network RP) to use, and reuse, for defining their own API versions that share a set of type definitions that is consistent across providers.",
```

Comment:
To avoid future maintainers modifying these definitions, will you be doing another update to consume common-types v5, and remove this reference, in e.g. next api version?

Reply:
I'll make the team aware of this so that this can be prevented/fixed in future api versions. Thanks Tim!